packages/vscode-extension/src/project/deviceSession.ts

```diff
@@ -63,29 +95,53 @@ export class DeviceSession implements Disposable {
   }
 
   constructor(
+    private readonly appRootFolder: string,
     private readonly device: DeviceBase,
-    private readonly devtools: Devtools,
-    private readonly metro: Metro,
     readonly dependencyManager: DependencyManager,
     readonly buildCache: BuildCache,
     debugEventDelegate: DebugSessionDelegate,
-    private readonly eventDelegate: EventDelegate
+    private readonly deviceSessionDelegate: DeviceSessionDelegate,
+    private readonly metroDelegate: MetroDelegate,
+    private readonly toolsDelegate: ToolsDelegate
   ) {
-    this.buildManager = new BuildManager(dependencyManager, buildCache);
+    this.deviceSettings = extensionContext.workspaceState.get(DEVICE_SETTINGS_KEY) ?? {
+      appearance: "dark",
```

Comment:
the default settings are not duplicated here and in project, is this intentional?